development-workflow.md

```diff
@@ -0,0 +1,61 @@
+# MeiliSearch Development Workflow
+
+MeiliSearch is an open soure project. 
+
+This document provides a general overview of the workflow for its development. 
+
+This document covers only the workflow for the [main project](meilisearch/MeiliSearch).
+
+[TOC]
+
+## Governance
+
+### MeiliSearch and Meili (the company)
+MeiliSearch is supported and developed mostly by Meili, the company. The project aims to be more and more managed by the community at large. Teams membership are, in theory, open to anyone working, or not, for Meili, the company.
+
+> __Disclaimer__
+> _The description of the teams and governance in this document are completely unrelated to how the company works and is organized internally._
+
+## 🔮 Roadmap
+The MeiliSearch roadmap is public on [roadmap.meilisearch.com](https://roadmap.meilisearch.com).
+Anyone can vote for future features or submit an new idea. The roadmap is moderated and prioritized by MeiliSearch CEO.
```

Comment:
In that case, should we mention that it's the Meili roadmap for MeiliSearch? That it does not include the community, because that statement make our project not much open source IMNSHO

Reply:
MeiliSeach is open source but not community-led. We are trying to gather and build a community around the open source project and we will take input from the community, that does not mean the community can will the roadmap for MeiliSearch. 

As I am writing this, I understand that I should rewrite the part about governance ^^

Reply:
Updated in next commit :)